scss/mixins/_utilities.scss

```diff
@@ -1,6 +1,20 @@
 // Utility generator
 // Used to generate utilities & print utilities
-@mixin generate-utility($utility, $infix, $is-rfs-media-query: false) {
+@mixin generate-utility($utility, $utility-name, $infix, $is-rfs-media-query: false) {
+  // Check Requires Fields
+
+  $required-fields: (
+    "property",
+    "values",
+  );
+
+  @each $field in $required-fields {
+    @include validate-utilities-field($utility, $utility-name, $field);
+  }
+
+  // End Check Requires Fields
+
+  $properties: map-get($utility, property);
```

Comment:
This is pretty nice but I'm wondering if we should not throw warning when we try to access those keys, rather than adding this whole part?

Reply:
These fields are required and I think there is no need to wait for them to be accessed. It will happen anyway